pyintacct/models/company.py

```diff
@@ -1,7 +1,20 @@
 from .base import API21Object
 from pydantic import BaseModel
-from typing import Optional
+from typing import Optional, List
 
+class AttachmentDetail(BaseModel):
+    attachmentname: Optional[str] = None
+    attachmenttype: Optional[str] = None
+    attachmentdata: Optional[str] = None
+
+class Attachment(BaseModel):
+    attachment: List[AttachmentDetail]
+
+class SupDoc(API21Object):
+    supdocid: Optional[str] = None
+    supdocfoldername: Optional[str] = None
+    supdocdescription: Optional[str] = None
+    attachments: Attachment = ...
```

Comment:
Something that might be nice is using default factories for the intermediary `Attachment` class, e.g.
```python
class Attachment(BaseModel):
    attachment: List[AttachmentDetail] = Field(default_factory=list)

class SupDoc(API21Object):
    supdocid: Optional[str] = None
    supdocfoldername: Optional[str] = None
    supdocdescription: Optional[str] = None
    attachments: Attachment = Field(default_factory=Attachment)
```
That way the `Attachment` and list don't need to be inserted manually.